### ISSUE TITLE
No Reserved Font Name for Source Han Serif license

### ISSUE THREAD

**Issue by NightFurySL2001** [adobe-fonts/source-han-serif#161]
In https://github.com/adobe-fonts/source-han-serif/blob/master/LICENSE.txt, there is no RFN for the name 'Source' and copyright clause. The RFN and copyright clause is included in [Source Han Sans](https://github.com/adobe-fonts/source-han-sans/blob/master/LICENSE.txt), [Source Han Mono](https://github.com/adobe-fonts/source-han-mono/blob/master/LICENSE.md), [Source Sans](https://github.com/adobe-fonts/source-sans/blob/release/LICENSE.md) and [Source Serif](https://github.com/adobe-fonts/source-serif/blob/release/LICENSE.md).

```
Copyright 2014-2021 Adobe (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe in the United States and/or other countries.
```

**Comment by NightFurySL2001** [comment]
P/S: is there any consideration for Adobe to release the RFN on 'Source'? It is limiting all modified versions from just converting the font from OTF to TTF as renaming will be required in the process of generating the modified version according to SIL OFL.

**Comment by punchcutter** [comment]
Thanks for pointing that out. I updated the main LICENSE files. The old one will still be in the release downloads for now, but will be updated in the next release. There's no plan to release the RFN. 

Since there is now a variable TTF I'm hoping that we can pull instance TTFs from that fairly easily. The last time I tried the tools failed, but now fontTools.varLib.instancer seems to work ok. I don't want to make the releases so much bigger, but that might be an option.
